src/osemgrep/targeting/dune

```diff
@@ -0,0 +1,12 @@
+; Various operations having to do with selection of files in Semgrep.
```

Comment:
ideally we can also put this out of osemgrep in libs/gitignore/ and release as a separate OPAM library
at some point.